CONTRIBUTING.md

```diff
@@ -94,50 +94,56 @@ The process of proposing a change to Talawa Admin can be summarized as:
          npm run test --watchAll=false --coverage
       ```
       
-      2. Debug Tests in browser 
+      2. Debug tests in browser 
+   
+      You can see the output of failing tests in broswer by running `jest-preview` package before running your tests
+
       ```
          npm install --legacy-peer-deps
          npm run jest-preview
          npm run test --watchAll=false --coverage
       ```
       
-      The preview for failed tests will be visible at http://localhost:3336 
+      The preview for failed tests will automatically be visible at `http://localhost:3336`, as shown below -
+
+      <img width="1904" alt="Debug Test Demo" src="https://user-images.githubusercontent.com/28570857/231282474-5ce3d47b-cba2-469d-891a-408e1b18c060.png">
```

Comment:
Add this image to the repo. We need to make sure that it's always available to the user even when they are working offline.

Reply:
Done, the image is now part of the repi ✅